utils/k8s/k8s.go

```diff
@@ -660,3 +660,26 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// CreatePersistentVolumeClaim is use to create the PVC in the given name space
+func (k8s K8S) CreatePersistentVolumeClaim(persistentVolumeClaim *core_v1.PersistentVolumeClaim, ns string) (result *core_v1.PersistentVolumeClaim, err error) {
+	persistentVolumeClaimNameSpace := k8s.Clientset.CoreV1().PersistentVolumeClaims(ns)
+	return persistentVolumeClaimNameSpace.Create(persistentVolumeClaim)
+}
+
+// ListPersistentVolumeClaim is used to list all the PVC in that name space
+func (k8s K8S) ListPersistentVolumeClaim(namespace string, opts *meta_v1.ListOptions) (result *core_v1.PersistentVolumeClaimList, err error) {
+	persistentVolumeClaimNameSpace := k8s.Clientset.CoreV1().PersistentVolumeClaims(namespace)
+	return persistentVolumeClaimNameSpace.List(*opts)
+}
+
+// ListSinglePersistentVolumeClaim is used to list single PVC in that name space
+func (k8s K8S) ListSinglePersistentVolumeClaim(namespace string) (result *core_v1.PersistentVolumeClaimList, err error) {
```

Comment:
Function name should be `GetPersistentVolumeClaim`
As in 
https://github.com/openebs/CITF/pull/53/files#diff-0838b5a1a898271846f11064eaefaab7R46